spec/services/remark_generators/frequency_checker_spec.rb

```diff
@@ -116,5 +116,38 @@ module RemarkGenerators
         end
       end
     end
+
+    context "employment_payments" do
```

Comment:
i can see these context names are consistent with the existing stuff...either way

```suggestion
    context "when checking employment_payments" do
```
or similar ^^